src/config.js

```diff
@@ -22,16 +22,6 @@ const DEFAULT_USERSYNC = {
   syncDelay: 3000
 };
 const DEFAULT_TIMEOUTBUFFER = 200;
-const DEFAULT_S2SCONFIG = {
```

Comment:
Shouldn't we keep default values here in this file ? 
How will other s2s adapters get default values ? Like this guy https://github.com/prebid/Prebid.js/pull/1819

Reply:
Makes sense to me @jaiminpanchal27, but want to let @snapwich weigh in before changing it.

Reply:
Also, we shouldn't have real defaults for `endpoint` and `syncEndpoint`.  We should be bidder agnostic.

Reply:
possibly in 1.0. If we omit them now it will break publishers integrations.

Reply:
I figured the adapters would specify their defaults.  Do we want to constrain server adapters to the same configuration?  We could, but I assumed we'd let each server adapter behave how it wished, as well as encapsulate as much code as possible inside the module as to not pollute the prebid-core w/ stuff related to a specific module.

Reply:
yeah I'm ok with each server adapter specifying it's own defaults.